README.md

```diff
@@ -1,64 +1,66 @@
+![screenshot](screenshot0.png?raw=true "A Debug-rendered vehicle composed of parts flying over a planet.")
+
+***This project is still deep in the pre-release development phase***
+
+OpenSpaceProgram is an open source initiative with the goal of creating a space flight simulator similar to Kerbal Space Program. This project also works as a general-purpose library for space games and simulations with very large universes.
+
+Written in C++17, this project mainly features a custom game engine and a universe/orbit simulator both relying on [EnTT](https://github.com/skypjack/entt/) and [Magnum](https://github.com/mosra/magnum). The universe and game engine are synchronized for a seamless spaceflight experience from a planet's surface to deep space.
+
+By taking advantage of Entity Component System (ECS) architectures and Data-Oriented Design, this project achieves simplicity, flexibility, low coupling, and excellent performance. With these techniques in action, we can easily avoid spaghetti code and optimize for a high part count.
+
+## Features
+
+### Core
+
+* Universe
+  * Support for 'infinite' universe sizes using hierarchical Coordinate Spaces
+  * Optimal data layout to support fast N-body and patched conics simulations
+  * 64-bit integers for coordinates
+* "ActiveScene" Game Engine
+  * Scene Graph
+  * Physics interface agnostic of physics engine
+  * Configurable multipass Renderer
+  * Wiring/Connection System
+    * Virtual control systems for vehicles (routable user inputs, PID, auto-landing, ...)
+    * Resource flow
+* Asset management
+  * Uses Standard glTF as a part model format
 * Can extend bulleted lists
 
-## Things to implement (roughly in order):
-* ~~Hello World, tests, and working build system~~
-* ~~Simple console interface for streamlining~~
-* ~~Minimal Universe of Satellites~~
-* ~~Showing glTF sturdy models~~
-* ~~ActiveAreas that load nearby satellites~~
-* ~~Intergrate Newton Dynamics~~
-* ~~Controllable debug crafts that can fly around~~
-* Terrain maybe?
-* Scripting or dynamic plugin interface?
+### Extra
+
+* Newton Dynamics Physics Engine integration
+* Rockets, RCS, and Fuel tanks
+* Rocket exhaust plume effects 
+* Icosahedron-based Planet surfaces with Level-of-detail subdivision
+
+### Test Application
+
+To act as a temporary menu and scenario loader, a console-based test application is implemented. It is **written in simple C++**, making it an excellent start to understanding the rest of the codebase.
+
+## Building
+
+[Build instructions](docs/building.md)
+
+If you just want to test out the project so far, then see the **Actions** tab to obtain automated builds for Linux or Windows.
+
+## Contributing
+
+Our development team is very small right now. We definitely need more crew to help to launch this project to its first release. Join our [Discord Server](https://discord.gg/7xFsKRg) for the latest discussions, and note that not everyone needs to be professional C++ developer to be involved.
+
```

Comment:
Maybe say:

"You don't need to be a professional C++ developer to be involved".

When I said "Note that not", i originally skipped ove rthe "not" and thought you were saying everyone needed to be a pro.

Reply:
I think you can reword it with all you need is passion, and we have a helpful community because saying 'professional C++ developer' sets pretty high standards, but this does not really have one.